MaterialDesignThemes.Wpf/SnackbarMessageQueueItem.cs

```diff
@@ -47,8 +48,37 @@ internal class SnackbarMessageQueueItem
         public bool IsPromoted { get; }
 
         /// <summary>
-        /// Still display this message even if it is a duplicate.
+        /// Always show this message, even if it's a duplicate
         /// </summary>
-        public bool IgnoreDuplicate { get; }
+        public bool ShowAlways { get; }
+
+        /// <summary>
+        /// Last time this message was shown
+        /// </summary>
+        public DateTime LastShownAt { get; set; }
+
+        public bool MessageExpired()
+        {
+            if (LastShownAt == null)
```

Comment:
`LastShownAt` is a value type, and will never be `null`. Perhaps make `LastShownAt` nullable

Reply:
Correct, i missed that. I think either replacing the null check with "LastShownAt == default(DateTime)" or just removing it would work. The default value of DateTime would produce correct results anyway (excluding extreme edge cases), your thoughts?

Reply:
Yea I am ok simply removing it.